extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -290,6 +290,20 @@ export class SubscriptionService {
       return this.timer_
         .timeoutPromise(timeout, subscriptionPlatform.getEntitlements())
         .then(entitlement => {
+          if (
+            entitlement.granted &&
+            this.cryptoHandler_.isDocumentEncrypted() &&
+            !entitlement.decryptedDocumentKey
+          ) {
+            const x =
```

Comment:
Let's come up with a better name than `x`. I know I suggested it in demos, but I was just being brief :)

Reply:
Changed to userOrDevLog.